src/net.cpp

```diff
@@ -2519,7 +2519,7 @@ void CConnman::Interrupt()
     }
 
     if (semMasternodeOutbound) {
-        for (int i=0; i<MAX_OUTBOUND_MASTERNODE_CONNECTIONS; i++) {
+        for (int i=0; i<(fMasternodeMode ? MAX_OUTBOUND_MASTERNODE_CONNECTIONS_ON_MN : MAX_OUTBOUND_MASTERNODE_CONNECTIONS); i++) {
```

Comment:
Heh :) let's maybe move this thing out of the loop completely? i.e.
```suggestion
        int nMaxMasternodeOutbound = fMasternodeMode ? MAX_OUTBOUND_MASTERNODE_CONNECTIONS_ON_MN : MAX_OUTBOUND_MASTERNODE_CONNECTIONS;
        for (int i = 0; i < nMaxMasternodeOutbound; i++) {
```